packages/celery-library/src/celery_library/worker/heartbeat.py

```diff
@@ -0,0 +1,21 @@
+import tempfile
+import time
+from pathlib import Path
+from typing import Final
+
+HEARTBEAT_FILE: Final[Path] = Path(tempfile.gettempdir()) / "celery_heartbeat"
+
+def update_heartbeat() -> None:
+    tmp_file = HEARTBEAT_FILE.with_suffix(".tmp")
+    tmp_file.write_text(f"{time.time()}")
```

Comment:
Q: would it be simpler to simply `touch` the file and use the modified timestamp as "heartbeat" time?